Example/Bellerophon/AppDelegate.swift

```diff
@@ -15,12 +15,23 @@ import Bellerophon
 class AppDelegate: UIResponder, UIApplicationDelegate, BellerophonManagerProtocol {
 
     var window: UIWindow?
-
+    var killSwitchURL: String?
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-        // Override point for customization after application launch.
+
+        killSwitchURL = "http://qa.cms.prolific.io/killswitch/status/vandelay/ios"
+
         BellerophonManager.sharedInstance.delegate = self
-        BellerophonManager.sharedInstance.killSwitchView = BellerophonHelperMethods.defaultBellerophonView("The app is killed", image: UIImage(named: "bellerophon.jpg")!)
+
+        let screenSize = UIScreen.mainScreen().bounds.size
+        let view = UIView(frame: CGRectMake(0, 0, screenSize.width, screenSize.height))
+
+        let imageView = UIImageView(frame: view.frame)
+        imageView.contentMode = .ScaleAspectFit
+        imageView.image = UIImage(named: "bellerophon.jpg")!
+        view.addSubview(imageView)
+
+        BellerophonManager.sharedInstance.killSwitchView = view
```

Comment:
Can you also add a label displaying some userMessage in the view? 

Like In `func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?)` 
or `func bellerophonStatus(manager: BellerophonManager, completion: (status: BellerophonObservable?, error: NSError?) -> ())`. 
It can also be placed in `func bellerophonWillEngage(manager: BellerophonManager)`